README.md

```diff
@@ -19,3 +19,50 @@ When a request is forwarded from Kong it adds a few headers when the app-key has
 2. X-Consumer-Custom-ID: The custom_id of the Consumer (if set) --> This is only set on system-consumers (e.g. the learningpath-frontend consumer)
```

Comment:
Enig. `Build & Run` trenger vi ikke. Forøvrig artig. Har ikke fått med meg `container:start`.
Jeg tenker egentlig at `About headers` ikke trengs lengre heller. Den var relevant tidligere da alle brukerene våre var definert av Kong.